pkg/k8s/kubectl.go

```diff
@@ -14,7 +14,7 @@ import (
 )
 
 type Kubectl interface {
-	Version() ([3]int, error)
+	Version() (*VersionInfo, error)
```

Comment:
Does this need to be a pointer? It doesn't really matter in this particular example, but as a rule of thumb copying data seems to be a better way to go about go: https://segment.com/blog/allocation-efficiency-in-high-performance-go-services/